docs/actions.md

```diff
@@ -313,20 +313,20 @@ Action | Description
 <a href="/actions/build_and_upload_to_appetize">build_and_upload_to_appetize</a> | Generate and upload an ipa file to appetize.io
 <a href="/actions/update_icloud_container_identifiers">update_icloud_container_identifiers</a> | This action changes the iCloud container identifiers in the entitlements file
 <a href="/actions/sh">sh</a> | Runs a shell command
-<a href="/actions/min_fastlane_version">min_fastlane_version</a> | Verifies the minimum fastlane version required
+<a href="/actions/setup_travis">setup_travis</a> | Setup the keychain and match to work with Travis CI
```

Comment:
even worse noise

Reply:
Are these randomized every time?

Reply:
yep, each docs PR has a slightly different order of things